packages/input-datepicker/test/lion-input-datepicker.test.js

```diff
@@ -493,6 +493,9 @@ describe('<lion-input-datepicker>', () => {
         expect(myOverlayOpenedCbHandled).to.be.true;
         await myElObj.selectMonthDay(1);
         expect(myUserSelectedChangedCbHandled).to.be.true;
+        /*         myElObj.overlayEl.hidden = true;
```

Comment:
I think you forgot to clean up this comment? :P

Reply:
Yes, I did! Made new commit to resolve.